inst/csv/replacementPatterns.csv

```diff
@@ -247,4 +247,50 @@ From,To,Pattern,Replacement
 "sql server","impala",".dbo.","."
 "sql server","impala","##","<hash><hash>"
 "sql server","impala","#","%temp_prefix%%session_id%"
-"sql server","impala","<hash><hash>","##"
\ No newline at end of file
+"sql server","impala","<hash><hash>","##"

```

Comment:
Ah, this was my bad too, it was a line change in this PR, but only because new lines were added after that line for the netezza changes.  Sorry @gowthamrao!  I don't think you have a problem here. I followed up with @tomwhite in a separate issue related to the impala translation.